gpcontrib/gp_orca/Makefile

```diff
@@ -0,0 +1,19 @@
+MODULE_big = gp_orca
+
+OBJS = gporca.o \
```

Comment:
```suggestion
OBJS = gporca.o
```

Reply:
In ADCC we  leave `\` near the last value. It allows to reduce the number of modified lines on the next commit. I suggest to follow here the same way.